polling_stations/apps/addressbase/management/commands/create_uprn_council_lookup.py

```diff
@@ -57,7 +58,7 @@ def handle(self, *args, **kwargs):
                     ) 
                 TO STDOUT
                 
-            with CSV HEADER DELIMITER ',';
+            with DELIMITER ',';
```

Comment:
this will close https://github.com/DemocracyClub/UK-Polling-Stations/issues/2713 :)